Update Console docs re: inviting users (#4762)
pulumi/docs: content/docs/intro/console/collaboration/organization-roles.md

> This only applies to newer organizations on the per-member subscription plan.
> Organizations billed per stack have slightly different rules regarding membership.

> To become a member of a Pulumi organization, you can be invited by an existing Pulumi
> organization administrator, or submit a request that the administrator can approve. However, depending on the [organization type]({{< relref "organizations#organization-kind" >}}), you must also be a member of the third-party organization or group backing the Pulumi
> organization.

Reviewer: Minor wording-tweak suggestion:

```suggestion
To become a member of a Pulumi organization, you can either be invited by an administrator of the organization or request to join the organization by navigating to it directly. Depending on the organization type]({{< relref "/docs/intro/console/organizations#organization-kind" >}}, you may also need to belong to the third-party organization or group that backs the Pulumi organization.
```

> To become a member of a Pulumi organization, you can be invited by an existing Pulumi
> organization administrator, or submit a request that the administrator can approve. However, depending on the [organization type]({{< relref "organizations#organization-kind" >}}), you must also be a member of the third-party organization or group backing the Pulumi
> organization.

Reviewer 2: We should make the clauses in this sentence more parallel. 

Also, it felt like the wording in the last sentence implies additional constraints on membership requirements, rather than a contradictory statement or one where we're contrasting something said earlier.

```suggestion
To become a member of a Pulumi organization, you must be invited by an existing Pulumi
organization administrator, or you must submit a request to the administrator for approval. In addition, depending on the [organization type]({{< relref "organizations#organization-kind" >}}), you must also be a member of the third-party organization or group backing the Pulumi
organization.
```

Author: That makes sense, I'll update, thank you!
